src/styles/star.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background: url(../images/star.svg) no-repeat center;
+    display: flex;
+
+    &:nth-child(-n + 4) {
```

Comment:
The selector `&:nth-child(-n + 4)` will apply the style to the first four children. Ensure this is the intended behavior, as it might be confusing. If you want to apply the style to a specific number of stars, consider using a more explicit selector.